cmd/example-gcp/main.go

```diff
@@ -50,7 +51,10 @@ func main() {
 		Bucket:    *bucket,
 		Spanner:   *spanner,
 	}
-	storage, err := gcp.New(ctx, gcpCfg, tessera.WithCheckpointSignerVerifier(signerFromFlags(), nil))
+	storage, err := gcp.New(ctx, gcpCfg,
+		tessera.WithCheckpointSignerVerifier(signerFromFlags(), nil),
+		tessera.WithBatching(1024, time.Second),
```

Comment:
If `WithBatching` is not defined, the queue will fail. Are we supposed to have default values in the storage implementation? Is this a mandatory option for personality to implement?

Reply:
You could set the default buffer size to 1 here, https://github.com/transparency-dev/trillian-tessera/blob/main/log.go#L43, and it should then flush for every new entry, i.e not batch.

Reply:
Yeah, exactly - that's what that little `!= nil` hook is intended for, I just didn't get around to adding _default_ defaults yet :) 

This PR seems as good a place as any, I'll add that now.

Thanks both!

Reply:
Ah, I've added some default values in MySQL storage implementation. I can remove them after this PR is merged.

https://github.com/transparency-dev/trillian-tessera/blob/8c5a7eef32de27b60645293dce813ca104ec70cf/storage/mysql/mysql.go#L58-L61